services/src/main/java/io/scalecube/services/ServiceInfo.java

```diff
@@ -0,0 +1,79 @@
+package io.scalecube.services;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * Helper class used to register service with tags as metadata in the scalecube cluster.
+ * parsing from service info to json and back.
+ */
+public class ServiceInfo {
+
+  @JsonIgnore
+  private static final ObjectMapper mapper = new ObjectMapper();
```

Comment:
i was trying to avoid manual parsing inventing format is error prone and a lot of "dirty" code 
as inventing a format is easy... dealing with it is not so nice